src/main/java/uk/gov/hmcts/reform/slc/services/steps/sftpupload/FtpClient.java

```diff
@@ -112,6 +112,10 @@ public void upload(PdfDoc pdfDoc) {
         });
     }
 
+    public boolean isHealthy() {
+        return runWith(sftpClient -> true);
+    }
```

Comment:
Looks like this method never returns `false` (it throws an exception).
Either change signature or catch it.

Reply:
+1 for catching exception and returning a result based on that - it's going to be more explicit